docs/manual/admin.md

```diff
@@ -1,11 +1,16 @@
-Redis Administration
-===
+---
+title: Redis Administration
+linkTitle: Redis Administration
+weight: 1
+aliases:
+  - /topics/admin
+---
+
 
 This page contains topics related to the administration of Redis instances.
 Every topic is self contained in form of a FAQ. New topics will be created in the future.
 
-Redis setup hints
------------------
+## Redis setup hints
```

Comment:
These two are equivalent in markdown. We try to avoid unnecessary changes because it makes the diff larger and messes up the git log and blame information.

Reply:
Thanks for the feedback and very valid point, @zuiderkwast. It's true that these two heading types are equivalent, but the underline-style headings only support H1 and H2 elements. As part of this organizational effort, we'd like to support H3, and even H4, headings, which means that we'll also eventually have to introduce the "#"-style headings. This sets us up for consistency. Hopefully this is only a minor pain in the short term.